server/controllers/linkVote.js

```diff
@@ -27,6 +27,35 @@ var castVote = function(req, res, next) {
   });
 };
 
+/**
+ * GET /vote/user
+ * Read users guide data
+ */
+var getLinkVoteByUser = function (req, res, next) {
+	LinkVote.findAll({
+		where: {
+			// need to have front end send userId from localStorage
+			userId: req.headers.userid
```

Comment:
Based on the property in the header, looks like this should be .userId (capital I)